cmd/kpod/import.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/projectatomic/libpod/libpod"
+
+	"github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	importFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "change, c",
+			Usage: "Apply imgspecv1 configurations to the created image",
+		},
+		cli.StringFlag{
+			Name:  "message, m",
+			Usage: "Set commit message for the image imported",
```

Comment:
`Set commit message for imported image`
Reads better

Reply:
fixed